revscoring/features/wikitext/features/tokenized.py

```diff
@@ -1,13 +1,15 @@
 from revscoring.datasources.meta import dicts, filters, mappers
-
 from ...meta import aggregators
+from . import base
 
 
-class Revision:
-
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+class Revision(base.BaseRevision):
 
+    def __init__(self, name, revision_datasources, tokens_datasource=None):
+        super().__init__(name, revision_datasources)
+        if tokens_datasource is None:
+            self.cjk = Revision(name, revision_datasources.cjk, tokens_datasource='CJK')
```

Comment:
Change `name` to `name` + ".cjk"

This will fix any name clashes in dependency space.

Reply:
I updated the name, but the test was failing due to other issue. In recently updated deltas package I adjusted regexp for numbers. I had to adjust it again:
from
number = r'[0-9][0-9\.\,]*(e[0-9]+)*|[\.\,][0-9][0-9\.\,]*(e[0-9]+)*'
to
number = r'[0-9]+([\,\.][0-9]+)*(e[0-9]+)*|([\.\,][0-9]+)+(e[0-9]+)*'

Explanation:
in this revscoring test:
'''
    cache = {p_text: "This is 45 72 tokens 23 72.",
             r_text: "This is 45 72 hats pants 85 72 72."}
    assert (solve(diff.datasources.number_delta, cache=cache) ==
            {'72': 1, '23': -1, '85': 1})
    assert (solve(diff.datasources.number_prop_delta, cache=cache) ==
            {'72': 1 / 3, '23': -1, '85': 1})
'''
The 72. was not tokenized as ['72',number, '.' punctuation] but as ['72.', etc] and it was not included in number_frequency feature computation:
'''
        self.number_prop_delta = frequencies.prop_delta(
            self.revision.parent.number_frequency,
            self.number_delta,
            name=self._name + ".number_prop_delta"
        )
'''